contracts/Registrar/PartnerRegistrar.sol

```diff
@@ -58,6 +58,8 @@ contract PartnerRegistrar is IBaseRegistrar, Ownable, IERC677TransferReceiver {
 
     function setFeeManager(IFeeManager feeManager) external onlyOwner {
         _feeManager = feeManager;
+
+        emit FeeManagerSet(address(this), address(_feeManager));
```

Comment:
The event should be emitted before the state change.

Reply:
👍🏾